src/scriptworker/artifacts.py

```diff
@@ -221,6 +223,16 @@ def get_artifact_url(context, task_id, path):
     return url
 
 
+# list_latest_artifacts {{{1
+async def list_latest_artifacts(queue, task_id, exception=TaskclusterFailure):
+    return await queue.listLatestArtifacts(task_id)
+
+
+async def retry_list_latest_artifacts(queue, task_id, exception=TaskclusterFailure, **kwargs):
+    kwargs.setdefault("retry_exceptions", tuple(set([TaskclusterFailure, exception])))
+    return await retry_async(list_latest_artifacts, args=(queue, task_id), kwargs={"exception": exception}, **kwargs)
```

Comment:
Do we actually need this?  https://github.com/taskcluster/taskcluster/blob/b09f8e56045981af8e7a163ea3e00b40f074108a/clients/client-py/taskcluster/client.py#L456 seems to have retry logic already.

Reply:
Ah, indeed. I'll remove this.